cpp/ql/src/Likely Bugs/Memory Management/NtohlArrayNoBound.qll

```diff
@@ -146,3 +146,19 @@ class NetworkToBufferSizeConfiguration extends DataFlow::Configuration {
     )
   }
 }
+
+module NetworkToBufferSizeConfiguration implements DataFlow::ConfigSig {
```

Comment:
This probably needs a QLDoc to make CI happy? And similarly with the `NetworkToBufferSizeFlow` module below.

Reply:
... and this might actually apply pretty widely. I'd be okay with no QLDocs on these, to be honest.

Reply:
If the CI is unhappy. I'll add them 😄

Reply:
Added them for the cases where the original class that I copied had one.

Reply:
I would make such config modules private.

Reply:
> I would make such config modules private.

Good point. I'll do that.

Reply:
Made private.